ui/src/components/admin/ModelsManager/ModelManagerController.js

```diff
@@ -115,350 +120,356 @@ Model table state transitions for each action:
 */
 const ModelManagerController = ({ baseUrl, cmsBase, children, ...props }) => (
   <NotificationsContext.Consumer>
-    {({ appendNotification, importProgress, setImportProgress }) => (
-      <Component
-        initialState={{
-          minRows: 0,
-          scrollbarSize: {
-            scrollbarWidth: 10,
-          },
-          filterValue: '',
-          data: [],
-          isLoading: false,
-          error: null,
-          rowCount: 0,
-          ...initPagingState,
-        }}
-        didMount={async ({ state, setState }) => {
-          try {
-            const [dataResponse, countResponse] = await loadData(baseUrl, state);
-            setState(() => ({
+    {({
+      appendNotification,
+      importProgress,
+      setImportProgress,
+      publishProgress,
+      setPublishProgress,
+    }) => (
+      <PublishNotificationsContext.Consumer>
```

Comment:
Unfortunately the `ModelManagerController` isn't using hooks, and due to time constraints I didn't have the bandwidth to convert it while working on this. So, had to wrap the whole thing in  `<PublishNotificationsContext.Consumer>` here, and most of the changes to this file are just indentation/spacing changes as a result.

I've commented around the blocks which are actual changes, the rest can be ignored.